### PR TITLE
Cloudflare Pages supports .node-version

### PR DESCRIPTION
**Pull request by bfdes**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,10 +25,11 @@ Version managers for Node.js which read a `.node-version` file include (in alpha
 
 Other products which test for `.node-version` include:
 
+- [cloudflare pages](https://developers.cloudflare.com/pages/platform/build-configuration#language-support-and-tools) Build fast sites. In record time.
```
Comment (bfdes) [inline README.md:28]: The real change.

File: README.md [hunk 2]
```diff
@@ -53,15 +54,15 @@ for users and especially Windows users to create a file compatible with your pro
 
 ## Compatibility Testing
 
-| Version Match | asdf | avn | direnv | fnm | n | nodenv | nodist  | nvm.fish | nvs |
```
Comment (bfdes) [inline README.md:56]: Formatting was caused by VS Code / Prettier. Very difficult to avoid.

Comment (shadowspawn) [reply README.md:56]: I have been caught by that too in projects where I have auto-save enabled, but found a convenient one-off work-around.

View > Command Palette... > File: Save without Formatting

File: README.md (hunk 1 above)
Comment (shadowspawn) [reply README.md:28]: Thanks 😄

**Comment by shadowspawn** [comment]
Added in direct commit, with you listed as co-author. Thanks for the PR.

**Pull request closed without merging**